### PR TITLE
Checklit Updated for correct guidelines

### PR DESCRIPTION
**Pull request by darkRaspberry**
Very Important regarding guidelines Please update

### CODE REVIEW

**Comment by puf17640** [comment]
I don't get the purpose of this PR

**Comment by Rocksus** [comment]
I believe it's for people who are doing
- [ x] on their checklists
instead of
- [x] doing it like this

Another way around is by telling them to tick the checkboxes after they've created the PR.

**Comment by puf17640** [comment]
> I believe it's for people who are doing
> 
> * [ x] on their checklists
>   instead of
> * [x]  doing it like this
> 
> Another way around is by telling them to tick the checkboxes after they've created the PR.

Oh yeah, didn't think that many would get it wrong

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ This pull request template helps you complete an application to the **Class of 2
 
 ## Checklist ✅
 
+<!--Please fill the checklist in correct way.
+- Wrong way []x [ x ] [x ] [ x] 
+- The only correct way is to to a small x between [x] like this.(No spaces between them.)
+Also
+- If you belong to 3/2 tier try checking the previous tier.
+- The PR should compulsory contain only 2 files. 1 is your display picture and 2 is your .md file.-->
+
```
Comment (juanpflores) [inline .github/PULL_REQUEST_TEMPLATE.md:13]:
```suggestion

[Reminder]: # (Fill the checklist below by adding an `x ` between the square brackets like this `[x]`. (No spaces between them)

```

Comment (juanpflores) [reply .github/PULL_REQUEST_TEMPLATE.md:13]: ❤️ this idea. This is what we can do

**Comment by darkRaspberry** [comment]
@juanpflores does check.

**Comment by darkRaspberry** [comment]
resolve #1598

**Comment by darkRaspberry** [comment]
This fixes #1598

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -4,6 +4,9 @@ This pull request template helps you complete an application to the **Class of 2
 
 ## Checklist ✅
 
+
+[Reminder]: # (Fill the checklist below by adding an `x ` between the square brackets like this `[x]`. No spaces between them. You can also tick the boxes after you have open the pull request.)
```
Comment (qaisjp) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: I think it should say `x` and not `x `. Also, `open` should be `opened`.

```suggestion
[Reminder]: # (Fill the checklist below by adding an `x` between the square brackets like this `[x]`. No spaces between them. You can also tick the boxes after you have opened the pull request.)
```

**Comment by darkRaspberry** [comment]
I think you should see the PR is not for adding list in year book. You have comments above. It's for updating guidelines. As u requested I have changed. Please merge.

**Comment by darkRaspberry** [comment]
@juanpflores @qaisjp please merge this. I have made requested changes.
This PR is not for adding name in yearbook. This PR is for updating guidelines.

**Comment by Rocksus** [comment]
Well it's a good thing knowing that they've automated the review process 😄

**Comment by darkRaspberry** [comment]
@Rocksus still not automated
contributor like us are making requesting changes.

**Comment by qaisjp** [comment]
Lol I don't know work for GitHub

**Pull request closed without merging**